### DISCUSSION TITLE
Ability to play more than one video files with smooth transaction

### DISCUSSION THREAD

**Discussion by blackaivengo** [CommunityToolkit/Maui#1058] [Q&A]
Hello.
I have the following scenario:
There is a page, that has some text and controls, and the video player is in the middle of it.
Upon page load, the video player needs to immediately start playing a video file.
When the first file is completed, we need to load the next video file, that will need to be played in a loop mode.
We do complete this using this control, but unfortunately, there is a problem.
 When the first video starts, right before it there is a black square shown instead of a video, and only then video is shown/ starts.
Same when the video is changed, there is a black flickering (probably a player black background), and then the second video is shown.

Any way to do so without seeing that black frame/background in between?

Thank you!

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
